addons/crm_iap_lead/models/crm_iap_lead_industry.py

```diff
@@ -8,11 +8,13 @@ class CrmIapLeadIndustry(models.Model):
     """ Industry Tags of Acquisition Rules """
     _name = 'crm.iap.lead.industry'
     _description = 'CRM IAP Lead Industry'
+    _order = 'sequence,id'
 
     name = fields.Char(string='Industry', required=True, translate=True)
     reveal_ids = fields.Char(required=True,
         help="The list of reveal_ids for this industry, separated with ','")
     color = fields.Integer(string='Color Index')
+    sequence = fields.Integer('Sequence')
```

Comment:
Isn't it any view to update it ? At least a technical tree view with a handle widget ?